plugins/utils/dashboard.nix

```diff
@@ -14,127 +14,370 @@ in {
+                types.str
+                "null"
+                "The icon of the module's section";
 
-            action = mkOption {
-              description = "Item action";
-              type = types.nullOr types.str;
-              default = null;
+              label =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The label of the module's section";
+
+              cwd_only =
+                helpers.defaultNullOpts.mkNullable
+                types.bool
+                "null"
+                "Whether to only change the directory when selecting a file";
             };
-          };
-        }));
+          });
         default = null;
+        example = {
+          limit = 10;
+          icon = "»";
+          label = "Most recently opened files";
+          cwd_only = true;
+        };
       };
 
-      sessionDirectory = mkOption {
-        description = "Path to session file";
-        type = types.nullOr types.str;
+      center = mkOption {
+        description = "Center section";
+        type = with types;
+          nullOr (listOf (submodule {
+            options = {
+              icon =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Item's icon";
+
+              icon_hl =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Icon's highlight group";
+
+              desc =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Item's description";
+
+              desc_hl =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Description's highlight group";
+
+              key =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Item's shortcut";
+
+              key_hl =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Shortcut's highlight group";
+
+              key_format =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Shortcut's text format, %s will be replaced with the value of `key`";
+
+              action =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "Item's action";
+            };
+          }));
         default = null;
+        example = [
+          {
+            icon = "Ω";
+            icon_hl = "Directory";
+            desc = "Find files";
+            desc_hl = "ModeMsg";
+            key = "<leader>f";
+            key_hl = "CurSearch";
+            key_format = "{ %s }";
+            action = "Telescope find_files";
+          }
+        ];
       };
```

Comment:
shouldn't all of this be in `config = { .... }` ? https://github.com/nvimdev/dashboard-nvim/blob/63df28409d940f9cac0a925df09d3dc369db9841/doc/dashboard.txt#L71